Code/GraphMol/DetermineBonds/DetermineBonds.cpp

```diff
@@ -0,0 +1,72 @@
+//
+//  Copyright (C) 2022 Sreya Gogineni and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "DetermineBonds.h"
+#include <GraphMol/RDKitBase.h>
+#include <YAeHMOP/EHTTools.h>
+#include <iostream>
+
+namespace RDKit {
+
+void connectivityHuckel(RWMol &mol, int charge) {
+    auto numAtoms = mol.getNumAtoms();
+    mol.getAtomWithIdx(0)->setFormalCharge(charge);
+    EHTTools::EHTResults res;
+    bool success = runMol(mol, res);
```

Comment:
`runMol()` currently always returns true (which should be fixed... but that's a different construction site). By ignoring the return value, you are assuming that it will always do so... 
It's probably good to document that assumption here:
```suggestion
    // as of this writing runMol() always returns true, so we ignore the return value.
    bool success = runMol(mol, res);
```